src/components/Checkbox/index.stories.js

```diff
@@ -1,86 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Checkbox from '.';
 
-const Template = (args) => <Checkbox {...args} />;
+const Template = (args) => {
+  const [checked, setChecked] = useState(false);
+  const onChange = (e) => {
+    setChecked(e.target.checked);
+  };
+  return <Checkbox onChange={onChange} checked={checked} {...args} />;
+};
 
 export const Primary = Template.bind({});
 Primary.args = {
```

Comment:
It would be good to add a short description for this story. Something along the lines of:

> `Checkbox` is fully controlled. Set the checked state with the `checked` prop, and use `onChange` to listen for user interaction.